src/elexmodel/handlers/s3.py

```diff
@@ -1,7 +1,14 @@
+import io
 import json
 import logging
+import queue
 
 import boto3
+import pandas as pd
+from botocore.session import get_session
+from dateutil import tz
+from s3transfer.manager import TransferManager
```

Comment:
Both s3transfer and botocore (which are the new imports) are requirements for boto3, so I think it should be OK!

Reply:
Ok, in general though, it's a good idea to have all required packages in `setup.py` or `requirements.txt` in case `boto3` drops this requirement in the future, or `boto3` switches to a version of `s3transfer` or `botocore` which breaks our code. Not a huge deal right now though, we can always add it later 😄

Reply:
I'm pretty sure our policy is top level requirements only